Jellyfin.Server.Implementations/Users/UserManager.cs

```diff
@@ -73,7 +76,8 @@ public UserManager(
             ILogger<UserManager> logger,
             IServerConfigurationManager serverConfigurationManager,
             IEnumerable<IPasswordResetProvider> passwordResetProviders,
-            IEnumerable<IAuthenticationProvider> authenticationProviders)
+            IEnumerable<IAuthenticationProvider> authenticationProviders,
+            UserMetrics? userMetrics = null)
```

Comment:
i am not too happy with this dependecy. I think it would be far better if we invert the responsibilities here or at the very least make it far more generic so that other code could also benefit from it